src/azure-cli-core/azure/cli/core/_credential.py

```diff
@@ -20,7 +20,7 @@ def _create_identity_credential(**kwargs):
         if not username or not authority or not tenant_id:
             raise CLIError("Missing username for user {}".format(home_account_id))
         auth_profile = AuthProfile(authority, home_account_id, tenant_id, username)
-        return InteractiveBrowserCredential(_profile=auth_profile, silent_auth_only=True,
+        return InteractiveBrowserCredential(profile=auth_profile, silent_auth_only=True,
```

Comment:
Use the right parameter name:

https://github.com/chlowell/azure-sdk-for-python/blob/cae2920fdc6f21cb8d112568fd9619200330c595/sdk/identity/azure-identity/azure/identity/_internal/msal_credentials.py#L87

```py
self._profile = kwargs.pop("profile", None)  # type: Optional[AuthProfile]
```